src/metricscollector/db/sqldb/sqldb_suite_test.go

```diff
@@ -0,0 +1,91 @@
+package sqldb_test
+
+import (
+	"database/sql"
+	"os"
+	"testing"
+
+	_ "github.com/lib/pq"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "metricscollector/db/sqldb"
+)
+
+var dbHelper *sql.DB
+
+func TestSqldb(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Sqldb Suite")
+}
+
+var _ = BeforeSuite(func() {
+	var e error
+
+	dbUrl := os.Getenv("DBURL")
+	if dbUrl == "" {
+		Fail("environment variable $DBURL is not set")
+	}
+
+	dbHelper, e = sql.Open(PostgresDriverName, dbUrl)
+	if e != nil {
+		Fail("can not connect database: " + e.Error())
+	}
+	cleanMetricsTable()
```

Comment:
Shouldn't we just be cleaning before each test?
It is fine here too but doesn't really help much.